request_handler.cc

```diff
@@ -110,14 +154,16 @@ void Response::AddHeader(const std::string& header_name, const std::string& head
 }
 
 int Response::GetStatus() {
```

Comment:
I didn't make that function signature (just changed return type), but you're right